docs/ios.md

```diff
@@ -5,110 +5,203 @@ permalink: /ios
+  * Importing (binary) MFC dump files
+  * Importing MIFARE Classic Tool (MCT) files
+  * "submit unknown stations" prompt is removed (only used by Brisbane Go Card and Troika)
+
+### UX differences
+
+* **Must press "scan card" on home screen**: iOS only allows scanning from a modal pop-up created
+  by CoreNFC.
+
+  By comparison, Android lets foreground applications scan at all times.
+
+* **Toasts are replaced with alert prompts.** Unfortunately, the user must acknowledge every toast
+  message.
+
+* **Launch from background is not supported**: not possible to implement in CoreNFC for the cards
+  we support -- only NDEF.
+
+* **Preferences screen misses detailed help text**: this doesn't exist on iOS.
+
+### UI differences
+
+* **Metrodroid interface follows iOS look and feel.** This is working as intended. ;)
+
+* **Card image and progress bar is not displayed while reading cards**: we can only display a
+  textual message using the CoreNFC API, and a percentage.
+
+* **Apple Maps used instead of Leaflet.**
+
+* **Theming follows system-wide preference only**: The "Farebot" theme is missing as a result (but
+  this would look very similar on iOS).
+
+* **Collapsed elements are not yet implemented.**  As a result:
 
   1. In subscription view the details are always shown
   2. In raw view subtrees are not collapsible
 
-## Accessibility
+### Accessibility
 
-Some acessibility features are omitted from iOS version.
+Some accessibility features are not available from iOS version:
 
-* TtsSpan that marks date/time and money text as such are omitted and you have
-to rely on your screen reader being smart enough.
+* `TtsSpan` that marks date/time and money text as such are omitted and you have
+  to rely on your screen reader being smart enough.
 
-* HiddenSpan used in time and route indications to provide readable alternative
-  to arrows is not implemented yet.
+* `HiddenSpan` used in time and route indications to provide readable
+  alternative to arrows is not implemented yet.
   
-## Other missing features
+### Other missing features
 
-* Locales without real translations are omitted: Irish, Maori, Malay and
-  Traditional Chinese.
+* Locales without real translations are omitted: Irish, Maori, Malay and Traditional Chinese.
   
 * Currently the library tests don't run due to a compiler problem.
 
 * Currently there are no iOS-specific tests.
 
-## System requirements
+## Build Metrodroid for iOS from source
 
-* iOS 13 beta or later
-* iPhone 7 or later
+**Note:** We're working on making this available via [TestFlight][].
 
-## Compilation
+Requirements:
 
-You need Android SDK due to technical reasons
-You also need Xcode 11 beta or later. Then:
+* macOS 10.14 Mojave or later
+* Android SDK (for technical reasons)
+* Xcode 11 beta or later
+* Java runtime environment
+
+**Note:** If you want to deploy your build to a physical device, you must also enroll in the
+[Apple Developer Program][apple-dev]. _This has an annual membership fee._
```

Comment:
Technically you can just disable NFC entitlements and import json on real device but probably no fun in this